docs_2.0/manual_EN/3.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -69,27 +69,28 @@ If you are reading this topic with the questions listed below, click them to jum
 
 4. Use nebula-console to connect to Nebula Graph.
 
-    [Nebula-console](https://github.com/vesoft-inc/nebula-console) is the native CLI client of Nebula Graph. In this step, Docker pulls the nebula-console images automatically from Docker Hub according to the path you set in the following commands and uses it to connect to the graphd service of Nebula Graph. You can use other clients to connect to Nebula Graph instead of Nebula-console, such as [Nebula Graph Studio](https://github.com/vesoft-inc/nebula-web-docker) and [clients for different programming languages](https://docs.nebula-graph.io/manual-EN/1.overview/2.quick-start/3.supported-clients/).
+    [Nebula-console](https://github.com/vesoft-inc/nebula-console) is the native CLI client of Nebula Graph. In this step, Docker pulls the nebula-console images automatically from Docker Hub according to the path you set in the following commands and uses it to connect to the Graph Service of Nebula Graph.
+    <!-- You can use other clients to connect to Nebula Graph instead of Nebula-console, such as [Nebula Graph Studio](https://github.com/vesoft-inc/nebula-web-docker) and [clients for different programming languages](https://docs.nebula-graph.io/manual-EN/1.overview/2.quick-start/3.supported-clients/). -->
 
     ```bash
     $ docker run --rm -ti --network nebula-docker-compose_nebula-net vesoft/nebula-console:v2-preview-nightly --address=graphd --port=3699 -u <username> -p <password>
     ```
 
     [TODO: Update the docker image tag on the release of 2.0 alpha.]
 
-    >**NOTE**: The default username is `root` and the default password is `nebula`.
+    >**NOTE**: By default, the authentication is disabled. Use any username or password to connect Nebula Graph. The default username is `root` and the default password is `nebula`.
 
     The following information indicates that you have connected to the Nebula Graph services:
 
     ```bash
-    Welcome to Nebula Graph (Version 5d10861)
+    Welcome to Nebula Graph v2.0.0-alpha!
 
-    (user@127.0.0.1) [(none)]>
+    (user@nebula) [(none)]>
     ```
 
-## Check the Nebula Graph service status and ports
+## Check the Nebula Graph service status and port
 
-Run `docker-compose ps` to list the service information of Nebula Graph, such as state and ports.
+Run `docker-compose ps` to list the service information of Nebula Graph, such as status and port.
```

Comment:
Run `docker-compose ps` to list all the services of Nebula Graph and their status and ports. ?